src/main/drivers/at32/serial_uart_at32f43x.c

```diff
@@ -315,7 +315,7 @@ uartPort_t *serialUART(UARTDevice_e device, uint32_t baudRate, portMode_e mode,
         if ((mode & MODE_TX) && txIO) {
             IOInit(txIO, OWNER_SERIAL_TX, RESOURCE_INDEX(device));
 
-            if ((options & SERIAL_INVERTED) == SERIAL_NOT_INVERTED) {
+            if (((options & SERIAL_INVERTED) == SERIAL_NOT_INVERTED) && !(options & SERIAL_BIDIR_PP_PD)) {
```

Comment:
It may be better to add new bit to options, signaling that `uartTxMonitor` shall be used